tests/LevelTest.php

```diff
@@ -7,17 +7,22 @@ class LevelTest extends \PHPUnit_Framework_TestCase
 {
     public function testLevel()
     {
-        $l = Level::TEST();
-        $this->assertNull($l);
+        try {
+            $level = Level::TEST();
+            $this->fail();
+        } catch (\Exception $exception) {
+            $this->assertTrue(true);
```

Comment:
@olafnorge this tests that an exception is thrown if a static method is called on `Level` that does not match any of the logging levels. Mostly this doesn't matter, as static methods are now deprecated.